Tests iOS/MyList/ArchiveTests.swift

```diff
@@ -206,7 +206,6 @@ extension ArchiveTests {
 
         app
             .webReaderView
-            .staticText(matching: "Hello, world")
```

Comment:
we can't test this anymore due to `url` being unique. Let me know what the best way to address this is

Reply:
it doesn't show the `hello` `html`, but let me investigate if we can import a different port or url to trigger this

Reply:
nvm, will update it!~